amworkflow/geometry/geometry.py

```diff
@@ -5,8 +5,7 @@
 from OCC.Core.StlAPI import StlAPI_Writer
 from OCC.Core.TopoDS import TopoDS_Shape
 from OCC.Extend.DataExchange import write_step_file, write_stl_file
-
-from amworkflow.geometry import composite_geometries, simple_geometries
+from amworkflow.geometry.simple_geometries import create_box
 
```

Comment:
I would like to stick with importing the whole module and then use function with module-name.function-name for better understanding which function comes from which module

Reply:
Furthermore I think that's the reason why the action are not working in the moment, because you changed that here but not in the function call (it's still simple_geoemtries.create_box). Please check